app/models/database.py

```diff
@@ -50,17 +50,7 @@ def __init__(self, config, path=APPLICATION_PATH_DATA):
             else:
                 logging.critical(f"Database {self.storage_type} not supported (only SQLite & PostgresSQL)")
 
-        # Use alembic commands rather than dropping into shell for running migrations
-        alembic_dir = os.path.join(APPLICATION_PATH, "alembic")
-        alembic_ini = os.path.join(APPLICATION_PATH, "alembic.ini")
-        alembic_cfg = AlembicConfig(alembic_ini)
-        alembic_cfg.set_main_option('script_location', alembic_dir)
-
-        # Define DB_URL envvar rather than using set_main_option so that command line
-        # migrations are still possible
-        os.environ["DB_URL"] = self.uri
-
-        command.upgrade(alembic_cfg, "head")
+        os.system(f"cd {APPLICATION_PATH}; DB_URL='{self.uri}' alembic upgrade head ")
```

Comment:
This is the reason why logs had disappeared.
As mentioned in [Alembic documentation](https://alembic.sqlalchemy.org/en/latest/api/config.html):

> When using programmatic configuration, make sure the env.py file in use is compatible with the target  configuration; including that the call to Python logging.fileConfig() is omitted if the programmatic configuration doesn’t actually include logging directives. 

So I reverted the change rather than trying to update `env.py`. This doesn't prevent unittests from passing.